gateway/src/apicast/policy/upstream/upstream.lua

```diff
@@ -61,7 +62,7 @@ function _M:content(context)
   local upstream = context[self]
 
   if upstream then
-    upstream:set_request_host()
+    upstream:rewrite_request()
```

Comment:
This is a bit confusing because the function in the call below, `upstream:call()`, already calls `rewrite_request()` at least in some cases.

Reply:
Should be fixed now.